### ISSUE TITLE
Line metrics incorrect with UTF-16 surrogate pairs

### ISSUE THREAD

**Issue by dzaima** [HumbleUI/Skija#9]
The code
```java
String[] families = {"DejaVu Sans Mono"};
FontCollection fc = new FontCollection();
fc.setDefaultFontManager(FontMgr.getDefault());
TextStyle ts = new TextStyle().setColor(0xffD2D2D2).setFontSize(20).setFontFamilies(families);
StrutStyle ss = new StrutStyle()
  .setFontFamilies(families)
  .setFontStyle(FontStyle.BOLD_ITALIC)
  .setFontSize(20)
  .setHeight(2)
  .setLeading(3)
  .setEnabled(true)
  .setHeightForced(true)
  .setHeightOverridden(true);
ParagraphStyle ps = new ParagraphStyle().setTextStyle(ts).setStrutStyle(ss);
ParagraphBuilder b = new ParagraphBuilder(ps, fc);
b.addText("𝕨𝕨𝕨𝕨𝕨");
Paragraph p = b.build();
b.close();
p.layout(Float.POSITIVE_INFINITY);
System.out.println(p.getLineMetrics()[0].getEndIndex());
```
prints `6`. But each `𝕨` is two `char`s, so the expected result is `10`.

64-bit Linux, Skija version 0.98.0

**Comment by dzaima** [comment]
[looks like](https://github.com/google/skia/commit/5dbdb1e75d6eff51254ae6c3979b5f70fc3a36b8) the skia function was changed to already return UTF-16 indices after Skija implemented it's own mapping.

**Comment by dzaima** [comment]
Can confirm that reverting [this](https://github.com/HumbleUI/Skija/commit/b703f6eb5dd709cf50b6ca2f854ffb5251dff6d4#diff-0e5cf00489d8ab43592919e2bd0962d54208a76b5d2a1f5a58fa92b833d3858dR128-R139) fixes that, and the line metrics work correctly with UTF-16.

**Comment by tonsky** [comment]
Fixed, should become available soon as 0.98.1
